doc/man1/flux-getattr.rst

```diff
@@ -6,28 +6,50 @@ flux-getattr(1)
 SYNOPSIS
 ========
 
-**flux** **getattr** *name*
+| **flux** **getattr** *name*
+| **flux** **setattr** *name* *value*
+| **flux** **lsattr** [*--values*]
 
-**flux** **setattr** *name* *value*
 
-**flux** **setattr** *name*
+DESCRIPTION
+===========
 
-**flux** **lsattr** [*--values*]
+Flux broker attributes provide a primitive configuration mechanism
+that predates :man1:`flux-config`.
```

Comment:
Is "a method for the broker to export information needed by Flux services and utilities." no longer true?  I think it'd be nice to say "it's mostly used for ..." now.

Reply:
also, i see its in the "SEE ALSO", but maybe a "see list of attributes in flux-broker-attributes(7)` might be nice.

Reply:
Since this was really about text that I changed in this PR, I'll go ahead and tack on a fix for this here.